YPP-0019.md

```diff
@@ -0,0 +1,228 @@
+
+These steps in 4 are encoded into the [addEthSeries.ts](https://github.com/yieldprotocol/environments-v2/blob/999c192ded2a4d8b658db6ab395659a9f35b727a/scripts/governance/addSeries/addEthSeries/addEthSeries.ts) scripts, while the steps 1 to 3 are the regular deployment scripts from the [environment deployment](https://github.com/yieldprotocol/environments-v2/tree/feat/eth-series/scripts/governance/newEnvironment).
+
+[This input](https://github.com/yieldprotocol/environments-v2/blob/999c192ded2a4d8b658db6ab395659a9f35b727a/scripts/governance/addSeries/addEthSeries/addEthSeries.mainnet.config.ts) will be used:
+
+## Base Config
+```
+export const assets: Map<string, string> = new Map([
+  [ETH, '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'],
+  [DAI, '0x6B175474E89094C44Da98b954EedeAC495271d0F'],
+  [USDC, '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'],
+  [WBTC, '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599'],
+  [WSTETH, '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0'],
+  [STETH, '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84'],
+  [LINK, '0x514910771af9ca656af840dff83e8264ecf986ca'],
+  [ENS, '0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72'],
+  [YVUSDC, '0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE'],
+  [UNI, '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'],
+])
+
+export const joins: Map<string, string> = new Map([
+  [ETH, '0x3bDb887Dc46ec0E964Df89fFE2980db0121f0fD0'],
+  [DAI, '0x4fE92119CDf873Cf8826F4E6EcfD4E578E3D44Dc'],
+  [USDC, '0x0d9A1A773be5a83eEbda23bf98efB8585C3ae4f4'],
+  [WBTC, '0x00De0AEFcd3069d88f85b4F18b144222eaAb92Af'],
+  [WSTETH, '0x5364d336c2d2391717bD366b29B6F351842D7F82'],
+  [LINK, '0xbDaBb91cDbDc252CBfF3A707819C5f7Ec2B92833'],
+  [ENS, '0x5AAfd8F0bfe3e1e6bAE781A6641096317D762969'],
+  [YVUSDC, '0x403ae7384E89b086Ea2935d5fAFed07465242B38'],
+  [UNI, '0x41567f6A109f5bdE283Eb5501F21e3A0bEcbB779'],
+])
+```
+
+## Proposal Config
+```
+export const governance: Map<string, string> = base_config.governance
+export const protocol: Map<string, string> = base_config.protocol
+export const assets: Map<string, string> = base_config.assets
+export const joins: Map<string, string> = base_config.joins
+export const newFYTokens: Map<string, string> = base_config.newFYTokens
+export const newPools: Map<string, string> = base_config.newPools
+export const newStrategies: Map<string, string> = base_config.newStrategies
+
+export const rateChiSources: Array<[string, string, string, string]> = [
+  [ETH, RATE, WAD.toString(), WAD.toString()],
+  [ETH, CHI, WAD.toString(), WAD.toString()],
+]
+
+// Assets that will be made into a base
+export const bases: Array<[string, string]> = [[ETH, base_config.joins.get(ETH) as string]]
+
+// Input data: baseId, ilkId, ratio (1000000 == 100%), line, dust, dec
+export const chainlinkDebtLimits: Array<[string, string, number, number, number, number]> = [
+  [ETH, DAI, 1400000, 1000000000, 1250000, 12],
+  [ETH, USDC, 1400000, 1000000000, 1250000, 12],
+  [ETH, ETH, 1000000, 2500000000, 0, 12], // Constant 1, no dust
+]
```

Comment:
Are these "line" in the billions?

Reply:
No, because it uses 12 decimals, in case we need to set a `dust` parameter below 1 ETH.